### DISCUSSION TITLE
Customising style of search table input

### DISCUSSION THREAD

**Discussion by markdynamo6** [filamentphp/filament#8418] [Help] [answered]
![image](https://github.com/filamentphp/filament/assets/63755857/130d6877-f96b-4a39-9b3f-e2cbf387bce5)

What is the recommended way to reinstate padding here?

**Comment by danharrin** [comment, marked as answer]
Upgrade Filament as there was a bug

**Reply by markdynamo6** [reply]
Awesome. Sorted:
![image](https://github.com/filamentphp/filament/assets/63755857/852df5b8-ed83-46f1-91c9-f68fd252b2e6)
